packages/database/src/api/Reference.ts

```diff
@@ -131,10 +138,11 @@ export class Reference extends Query {
     validateCallback('Reference.set', 2, onComplete, true);
 
     const deferred = new Deferred();
-    this.repo.setWithPriority(
+    repoSetWithPriority(
+      this.repo,
       this.path,
       newVal,
-      /*priority=*/ null,
+      null,
```

Comment:
nitpick - the comment still feels useful.

Reply:
Fixed